web/views.py

```
@@ -503,6 +503,18 @@ def profile(request):
 
     return render(request, "profile.html", context)
 
+@login_required
+def notification_preferences(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `notification_preferences`

(ANN201)